### ISSUE TITLE
Can't work with Microsoft Word

### ISSUE THREAD

**Issue by easypickings** [rxhanson/Rectangle#631]
- [ ] This is a new bug that hasn't been documented in a closed issue or in the Readme.

macOS version: macOS Monterey 12.0.1
Rectangle version: 0.49
Logs if applicable (In Rectangle menu, hold option, "View Logging..."): Unable to obtain accessibility element

Seems like #182, except I encountered it with Microsoft Word, and restarting the app can't fix 😢

**Comment by rxhanson** [comment]
Unfortunately, this is not something that can be fixed Rectangle side. You will have to file a bug with Microsoft, and the issue would be that Word windows are not responding to the macOS accessibility API for window positioning and sizing.
